spec/services/admin/current_teachers_spec.rb

```diff
@@ -1,13 +1,13 @@
 describe Admin::CurrentTeachers do
   describe '#current' do
-    context 'with appropriate body IDs' do
-      it 'returns teachers with ongoing induction periods for the given appropriate bodies' do
-        ab1 = FactoryBot.create(:appropriate_body)
-        ab2 = FactoryBot.create(:appropriate_body)
-        teacher1 = FactoryBot.create(:teacher)
-        teacher2 = FactoryBot.create(:teacher)
-        teacher3 = FactoryBot.create(:teacher)
+    let(:teacher1) { FactoryBot.create(:teacher) }
+    let(:teacher2) { FactoryBot.create(:teacher) }
+    let(:teacher3) { FactoryBot.create(:teacher) }
+    let(:ab1) { FactoryBot.create(:appropriate_body) }
```

Comment:
I know you just repositioned this into let blocks but my mind immediately thinks of Accredited body when I see AB, it would be nice to be more explicit.

```suggestion
    let(:appropriate_body) { FactoryBot.create(:appropriate_body) }
```

That would also mean you could omit the first one:

E.G

```
FactoryBot.create(
          :induction_period,
          teacher: teacher1,
          appropriate_body:)
```

Reply:
I fought the urge to do just that. I will. Good call.